onnx/common/tensor.h

```diff
@@ -27,6 +29,115 @@ struct Tensor final {
   bool is_raw_data_;
   std::string raw_data_;
 
+  template<typename T>
+  static void add_nums(T* x, const T* y) {
```

Comment:
There are std::plus, std::minus, std::multiplies and std::divides

Reply:
changed to use

Reply:
I guess these `*_nums` are not used anymore so they can be deleted?